src/main/resources/db/migration/V0001__email.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE email
```

Comment:
Flyway não se resume a script sequenciais e na verdade não se resume a script SQL.

* Existem script que rodam repetidamente. Esses começam com o sufixo `R`, e vão rodar sempre que a aplicação subir. Nunca usei, mas pode vir a ser útil.
* Você pode criar "scripts" em Java para rodarem antes do início do projeto. Isso pode ser útil para situações mais elaboradas.